README.md

```diff
@@ -1129,6 +1129,31 @@ providers:
         path: /folder/bar
 ```
 
+
+## ProcessEnv
+
+Load the environment variables from the parent process as needed.
+
+### Authentication
+No need.
+
+### Features
+
+* Sync - `yes`
+* Mapping - `yes`
+* Modes - `read`
+
+### Example Config
+
+```yaml
+providers:
+  process_env:
+    env_sync:
+
+    env:
+      ETC_DSN:
```

Comment:
From my understanding, this setup get the value `ETC_DSN` from env var
for example 
```sh
$ export ETC_DSN=teller-value
$ teller yaml
ETC_DSN: teller-value
```

It will be better if we also allow the user to control the key name.
for example
the key under env should be the teller key, and the value is the name of the env key that we want to pick up.
.teller config:
.teller config:
```yaml
providers:

  process_env:
    env:
       TELLER-KEY: ETC_DSN
```
Execute teller:
```sh
$ export ETC_DSN=teller-value
$ teller yaml
TELLER-KEY:: teller-value
```

Reply:
Yes, it'll use `field` to be consistent with other providers.  So:

```yaml
process_env:
  env:
    GH_USERNAME: # will use this field
    GH_TOKEN:
      field: TOKEN # will use this field to map to GH_TOKEN
```